homeassistant/components/skybell/binary_sensor.py

```diff
@@ -20,7 +20,7 @@
 
 from . import DEFAULT_ENTITY_NAMESPACE, DOMAIN as SKYBELL_DOMAIN, SkybellDevice
 
-SCAN_INTERVAL = timedelta(seconds=10)
+SCAN_INTERVAL = timedelta(seconds=90)
```

Comment:
The PR title talks about a "too short timeout".
The PR contents adjust the scan interval.

There is no description provided on how those relate or what the issue is that is being solved.

Could you update the PR description to provide context on this change?

Thanks!

../Frenck